src/main/java/org/eclipse/tractusx/puris/backend/controller/HealthController.java

```diff
@@ -0,0 +1,27 @@
+package org.eclipse.tractusx.puris.backend.controller;
```

Comment:
Add missing license header

Reply:
Added license header to the `HealthController`.